proj4/src/test/scala/Main.scala

```diff
@@ -0,0 +1,123 @@
+package dwins
+
+import scala.sys.process._
+
+object Proj {
+  implicit class withAsStream(val a: String) extends AnyVal {
+    def asStream: java.io.InputStream = 
+      new java.io.ByteArrayInputStream(a.getBytes(java.nio.charset.StandardCharsets.UTF_8))
+  }
+
+  def grid(minx: Int, miny: Int, maxx: Int, maxy: Int): Iterator[(Double, Double)] = 
+    for { 
+      x <- (minx to maxx).iterator
+      y <- miny to maxy
+    } yield (x.toDouble, y.toDouble)
+
+  def loan[T](r: => scala.io.Source)(f: scala.io.Source => T): T = {
+    val resource = r
+    try
+      f(resource)
+    finally
+      resource.close
+  }
+
+  def main(args: Array[String]): Unit = {
+    val registeredCodes = 
+      loan(scala.io.Source.fromInputStream(getClass().getResourceAsStream("/geotrellis/proj4/nad/epsg"))) { source =>
+        source.getLines
+          .filter { _ startsWith "<" }
+          .map { s => s.tail.take(s.indexOf('>') - 1) }
+          .to[Set]
+      }
+    val knownCodes = 
+      loan(scala.io.Source.fromFile("/Users/dwins/Projects/radiantblue/proj.4/nad/epsg")) { source =>
```

Comment:
Sorry, the above comment was about this line. `/Users/dwins/` probably shouldn't be there, so where should this file go?